src/DTF/Tools/SfxCA/SfxCA.cpp

```diff
@@ -107,33 +107,42 @@ int InvokeOutOfProcManagedCustomAction(MSIHANDLE hSession, const wchar_t* szEntr
         wchar_t szModule[MAX_PATH] = {0};
         GetModuleFileName(g_hModule, szModule, MAX_PATH);
 
-        const wchar_t* rundll32 = L"rundll32.exe";
+        wchar_t szSystemPath[MAX_PATH] = {0};
+        GetSystemDirectory(szSystemPath, MAX_PATH);
+
         wchar_t szRunDll32Path[MAX_PATH] = {0};
-        GetSystemDirectory(szRunDll32Path, MAX_PATH);
-        wcscat_s(szRunDll32Path, MAX_PATH, L"\\");
-        wcscat_s(szRunDll32Path, MAX_PATH, rundll32);
+        wcscat_s(szRunDll32Path, MAX_PATH, szSystemPath);
```

Comment:
At first, a little weird to concatenate to an empty string. But I decided I prefer the way this looks/works.